functions.sh

```diff
@@ -189,6 +189,21 @@ choose_all(){
     fi
 }
 
+# Changes the default Namespace, requires the `ns` plugin.
```

Comment:
[nit] we could avoid the need for `ns` by using `kubectl config set-context --current --namespace=`